src/components/GlobalPreferences/HelpAndFeedback/HelpAndFeedback.js

```diff
@@ -1,36 +1,57 @@
 import React, { useCallback } from 'react'
 import PropTypes from 'prop-types'
-import { Box, Switch, Info, Button, GU, textStyle, useTheme } from '@aragon/ui'
+import { Box, Switch, Info, Link, GU, textStyle, useTheme } from '@aragon/ui'
 import { useHelpScout } from '../../HelpScoutBeacon/useHelpScout'
 import helpAndFeedbackPng from './assets/help-and-feedback.png'
+import { useConsole } from '../../../apps/Console/useConsole'
 import { getAppPath } from '../../../routing'
 
-function HelpAndFeedback({ historyPush, onClose, locator }) {
+function HelpAndFeedback({ historyPush, locator, onClose }) {
   const theme = useTheme()
   const { optedOut, setOptedOut } = useHelpScout()
+  const {
+    optedOut: optedOutFromConsole,
+    setOptedOut: setOptedOutFromConsole,
+  } = useConsole()
+
   const handleOptOutChange = useCallback(() => setOptedOut(!optedOut), [
     setOptedOut,
     optedOut,
   ])
 
-  const handleGoToEmbeddedConsole = useCallback(() => {
+  const handleOptOutConsoleChange = useCallback(
+    () => setOptedOutFromConsole(!optedOutFromConsole),
+    [optedOutFromConsole, setOptedOutFromConsole]
+  )
+
+  const handleConsoleLinkClick = useCallback(() => {
     onClose()
     historyPush(
       getAppPath({
         dao: locator.dao,
         instanceId: 'console',
       })
     )
-  }, [onClose, historyPush, locator.dao])
+  }, [historyPush, locator, onClose])
 
   return (
     <>
       <Box heading="Help Scout">
+        <img
+          src={helpAndFeedbackPng}
+          alt="Help Scout"
+          css={`
+            display: block;
+            margin: 0 auto;
```

Comment:
From the designs, this looks like it should have quite a lot more padding (maybe 6 GU)?